src/main/java/com/facebook/openwifirrm/modules/ApiServer.java

```diff
@@ -424,6 +431,89 @@ private String getOpenApiJson(Request request, Response response) {
 		return Json.pretty(getOpenApi());
 	}
 
+	/** TODO [ZoneBasedRrmScheduling] remove this once we move to venues
```

Comment:
I think we should place `TODO`s outside of the javadoc, because javadoc is used to generate documents and it's not good to include development TODOs in the documentation. But maybe I'm wrong, feel free to push back. @elludraon

Reply:
I need to get used to javadoc, i think it makes sense to separate it.

Reply:
There should be some extensions/tools in your IDE to auto-generated a javadoc template for a method. Then you could fill in that template easily. Usually you are only expeced to add method description, @param, @return. For this PR, I ok that if you do it separately.